CMakeLists.txt

```diff
@@ -40,6 +40,7 @@ include(EthCcache)
 
 # Let's find our dependencies
 include(EthDependencies)
```

Comment:
Just a general remark (not something to fix in this PR): it seems that only boost is in `EthDependencies` now, everything else has its own file. We should consider moving it to a separate file and renaming `EthDependencies.cmake` to something more fitting. It would be easier to find that way.